### PR TITLE
Edit/postgres setup

### PR DESCRIPTION
**Pull request by Pma913**
# PR Template

### Types of changes made?

- [x]  javascript
- [x]  functionality

### What does this PR do?

- Install all dependencies needed: Knex, Postgres, Express, CORS, Nodemon
- Setup a script to be able to use command `npm start` in place of `nodemon server.js`
- Create local database and configure using knex library
- Migrate table schema to the database, and alter the table to update character limits
- Seed database with our data array
- Setup Express server
- Successfully test the connection between server & database in Postman

### Relevant Tickets?

- Finishes iteration or ticket for: N/A

### Questions/Comments/Relevant Screenshots?

When reporting a bug, include: N/A

** After this PR is merged, all teammates except Patrick will clone down the repo and setup their own local database using the same configs.

** We will work on deploying the database via Heroku later this week.

### CODE REVIEW

**Comment by tialaaa** [comment]
No notes to add since we all worked on this together. Great job troubleshooting through the migrations, team!

File: migrations/20230528200027_update.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * @returns { Promise<void> }
+ */
+exports.down = async function(knex) {
+  await knex.schema.alterTable('places', (table) => {
+    table.dropColumn('description');
+    table.dropColumn('src');
+    table.dropColumn('alt');
+  });
+  await knex.schema.alterTable('places', (table) => {
+    table.string('description');
```
Comment (niksseif) [inline migrations/20230528200027_update.js:24]: Nice job 🙌🏼 - This looks great.

File: knexfile.js [hunk 2]
```diff
@@ -8,10 +8,10 @@ module.exports = {
   development: {
     client: 'pg',
     connection: {
+      port: 5432,
       database: 'pet_furriendly_db',
       user:     'postgres',
-      password: 'Turing2301',
-      port: 5432
+      password: 'Turing2301'
```
Comment (niksseif) [inline knexfile.js:14]: For this project, it's okay to have the password showing up, but when we are working with real users, you might want to hide this password in the` .env` file.

File: migrations/20230528184154_init.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+  return knex.schema.createTable('places', function(table) {
+    table.increments('id');
```
Comment (niksseif) [inline migrations/20230528184154_init.js:7]: Nice work 🙌🏼

**Review by niksseif — COMMENTED** [review]
Nice work everyone.

**Pull request merged**